gpytorch/kernels/grid_interpolation_kernel.py

```diff
@@ -130,21 +129,26 @@ def _tight_grid_bounds(self):
     def has_custom_exact_predictions(self):
         return True
 
-    def _compute_grid(self, inputs):
+    def _compute_grid(self, inputs, batch_dims):
         batch_size, n_data, n_dimensions = inputs.size()
+        if batch_dims == (0, 2):
+            inputs = inputs.view(inputs.size(0), inputs.size(1), -1, 1)
+            inputs = inputs.transpose(1, 2).contiguous()
+            batch_size = batch_size * inputs.size(1)
+            n_dimensions = n_dimensions // inputs.size(1)
+
```

Comment:
Should we raise a NotImplementedError here if that check is False and `batch_dims` is something other than `None`? What's the desired behavior for other arguments?